src/main/java/hybrid/it/internship/library/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,5 @@
+package hybrid.it.internship.library.exception;
+
+public class EntityNotFoundException extends RuntimeException {
+
```

Comment:
Add a constructor with a string parameter so you can send some more info about the entity which is not found.